search/house_detector.cpp

```diff
@@ -1050,7 +1050,7 @@ void MergedStreet::FinishReadingHouses()
 
 HouseProjection const * MergedStreet::GetHousePivot(bool isOdd, bool & sign) const
 {
-  typedef my::limited_priority_queue<HouseProjection const *, HouseProjection::LessDistance> QueueT;
+  typedef base::limited_priority_queue<HouseProjection const *, HouseProjection::LessDistance> QueueT;
```

Comment:
`using Queue = base::limited_priority_queue<HouseProjection const *, HouseProjection::LessDistance>;`
?

Reply:
done, but this is out of the scope of the PR